pkg/entities/wallet.go

```diff
@@ -574,6 +575,27 @@ func (e *Entity) listSuiWalletAssets(req request.ListWalletAssetsRequest) ([]res
 			return nil, "", "", err
 		}
 
+		for index, asset := range assets {
+			searchQ := coingeckosupportedtokens.ListQuery{Symbol: strings.ToLower(asset.Token.Symbol)}
+			tokens, err := e.repo.CoingeckoSupportedTokens.List(searchQ)
+			if err != nil {
+				e.log.Fields(logger.Fields{"searchQ": searchQ}).Error(err, "[listSuiWalletAssets] repo.CoingeckoSupportedTokens.List() failed")
+				continue
+			}
+			for _, t := range tokens {
+				prices, err := e.svc.CoinGecko.GetCoinPrice([]string{t.ID}, "usd")
+				if err != nil {
+					e.log.Fields(logger.Fields{"id": t.ID}).Error(err, "[entity.SearchCoins] svc.CoinGecko.GetCoinPrice() failed")
+					continue
+				}
+				if t.Name == asset.Token.Name {
```

Comment:
this is token name comparison between coingecko and sui rpc right? is this always right?

Reply:
đúng rồi ạ, đây là cách tốt nhất để lọc token có cùng symbol ạ.